src/rewards/veterans.rs

```diff
@@ -0,0 +1,35 @@
+use serde::Deserialize;
+
+pub type VeteranAdvisorId = String;
+pub type VeteranAdvisorReward = rust_decimal::Decimal;
+
+pub type VeteranAdvisorRewards = Vec<(VeteranAdvisorId, VeteranAdvisorReward)>;
+
+#[derive(Deserialize)]
+pub struct VeteranReview {
```

Comment:
nit: naming is a bit confusing. This is not actually a review, but rather holds info about a vCA

Reply:
Indeed, let me find a more suitable naming for this.